src/world/Action/EffectBuilder.cpp

```diff
@@ -49,65 +42,79 @@ void EffectBuilder::moveToResultList( Entity::CharaPtr& chara, EffectResultPtr r
   it->second->push_back( std::move( result ) );
 }
 
-void EffectBuilder::heal( Entity::CharaPtr& effectTarget, Entity::CharaPtr& healingTarget, uint32_t amount, Common::ActionHitSeverityType severity, Common::ActionEffectResultFlag flag )
+void EffectBuilder::heal( Entity::CharaPtr& effectTarget, Entity::CharaPtr& healingTarget, uint32_t amount, Common::ActionHitSeverityType severity, Common::ActionEffectResultFlag flag, uint64_t resultDelayMs )
 {
-  EffectResultPtr nextResult = make_EffectResult( healingTarget, getResultDelayMs() );
+  EffectResultPtr nextResult = make_EffectResult( healingTarget, nullptr, Common::Util::getTimeMs() + resultDelayMs );
```

Comment:
why is this delay calculated manually each time?

Reply:
it allows applying different delay for individual result, which is need for some scripts to use like summoner dots